bitnami/apache/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: apache
-version: 7.2.14
+version: 7.2.15
```

Comment:
Bump the minor version instead.

Reply:
removed patch version update and updated minor version to 7.3.0